src/smt/smt_setup.cpp

```diff
@@ -177,7 +177,7 @@ namespace smt {
                  setup_QF_BVRE();
             else if (m_logic == "QF_AUFLIA")
                 setup_QF_AUFLIA(st);
-            else if (m_logic == "QF_S" || m_logic == "QF_SLIA")
+            else if (m_logic == "QF_S" || m_logic == "QF_SLIA" || m_logic == "QF_SLIA")
```

Comment:
Should be `QF_SNIA`? (`QF_SLIA` occurs twice)

Reply:
Yes,  fixed, it seems this piece of code was not reached in the benchmarks (hopefully).